openhands/server/data_models/resolver.py

```diff
@@ -0,0 +1,124 @@
+from typing import Any, Literal
```

Comment:
Actually, I think we can just remove this `data_models/resolver.py` file. We can just call the functions in `send_pull_request.py` and `resolve_issue.py` directly.